source/about.rst

```diff
@@ -3,57 +3,119 @@
 About
 #####
 
-The |CL| delivers a secure, hardware optimized OS. Its easy updates ensure that
-software dependencies remain mutually compatible.
-
-|CL| does this via custom infrastructure components and process innovations.
+|CL-ATTR| does things differently. Our software architecture provides a 
```

Comment:
This is a super-specific target audience declaration, can we make it a little bit broader? 

e.g. "compute, server, and cloud" is a lot more inclusive than "cutting-edge computation in the cloud" which is really specific.

Reply:
See latest commit.